repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/tracing/rewritetransid/system-model.cfg.xml

```diff
@@ -13,4 +13,5 @@
                       default="true"/>
         </destinations>
     </repose-cluster>
+    <tracing-header rewrite-header="true"/>
```

Comment:
Gotcha.

Yeah, that's why I would also be totally okay with making the new element required and keeping the enabled attribute. That way there is no confusion.

I still don't love the original solution, though.

Anyway, that's @adrianjgeorge call to make. :stuck_out_tongue: